web/src/components/__tests__/CategoriesToolbar.spec.tsx

```diff
@@ -0,0 +1,68 @@
+import { CategoryModel } from 'api-client'
```

Comment:
We renamed/moved the whole api-client, so you have to change this to
```suggestion
import { CategoryModel } from 'shared/api'
```